frontend/package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "dependencies": {
     "@craco/craco": "^6.1.1",
+    "@headlessui/react": "^1.0.0",
```

Comment:
I see we're adding it as a dependency, but the changes in package-lock.json are _tooo_ huge to be just one package's addition? Or is that the case? :/